src/main/java/org/elasticsearch/transport/TransportService.java

```diff
@@ -91,9 +91,6 @@ protected void doStart() throws ElasticsearchException {
         adapter.txMetric.clear();
         transport.transportServiceAdapter(adapter);
         transport.start();
-        if (transport.boundAddress() != null && logger.isInfoEnabled()) {
-            logger.info("{}", transport.boundAddress());
```

Comment:
but it wasn't replaced by any other similar line somewhere else? I think this line is quite important, as gets printed out at startup in the console by default like the following:

```
[2015-01-06 10:18:01,288][INFO ][transport                ] [Gertrude Yorkes] bound_address {inet[/0.0.0.0:9300]}, publish_address {inet[/192.168.0.27:9300]}
```

I don't think we should remove it.